.gitignore

```diff
@@ -8,3 +8,4 @@ dnf-json
 /.fleet
 /.idea
 /gen-oscap
+local.env
```

Comment:
nitpick, 1 newline at the end of each file

Reply:
ugh, new editor,  I've changed settings and fixed it :)